src/Microsoft.DotNet.GenAPI/Shared/Utils.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+
+namespace Microsoft.DotNet.GenAPI.Shared;
+
+/// <summary>
+/// Utility functions shared between CLI and MSBuild tasks frontends.
+/// </summary>
+public class Utils
```

Comment:
```suggestion
public static class Utils
```

Make it explicit that there's no point instantiating this class